### PR TITLE
make protobuf constraint less strict

### PR DESCRIPTION
**Pull request by eunwoosh**
This PR makes protobuf version constraint less strict.
CI result : https://ci-ote.iotg.sclab.intel.com/job/ote/job/pr-ote-test/365/

### CODE REVIEW

**Comment by eunwoosh** [comment]
run ote-test

File: external/anomaly/constraints.txt [hunk 1]
```diff
@@ -17,4 +17,4 @@ requests==2.26.0
 scikit-image==0.17.2
 scikit-learn==0.24.2
 torchmetrics==0.6.0
-protobuf==3.20.0
+protobuf<3.21.0
```
Comment (ashwinvaidya17) [inline external/anomaly/constraints.txt]: We are introducing conflicting changes in our PR. https://github.com/openvinotoolkit/training_extensions/pull/1124/files I'll update my PR to reflect your changes.

Comment (ashwinvaidya17) [reply external/anomaly/constraints.txt]: Also, I am not sure what constraints.txt does. Is it a right practice to add `<` or `>` in constraints.txt? And in that case, how is it different from `requirements.txt`?

Comment (eunwoosh) [reply external/anomaly/constraints.txt]: I see. I'll put it back as previous.
For second comment, as far as I know, constraints file literally just constrains package version during install.
So it doesn't make some package installed like requirements.txt. Please refer this [docs](https://pip.pypa.io/en/stable/user_guide/#constraints-files).

**Comment by eunwoosh** [comment]
run ote-test

**Review by goodsong81 — APPROVED** [review]

**Pull request merged**